internal/event/target/webhook.go

```diff
@@ -297,3 +296,23 @@ func NewWebhookTarget(ctx context.Context, id string, args WebhookArgs, loggerOn
 
 	return target, nil
 }
+
+// DrainBody close non nil response with any response Body.
+// convenient wrapper to drain any remaining data on response body.
+//
+// Subsequently this allows golang http RoundTripper
+// to re-use the same connection for future requests.
+func DrainBody(respBody io.ReadCloser) {
+	// Callers should close resp.Body when done reading from it.
+	// If resp.Body is not closed, the Client's underlying RoundTripper
+	// (typically Transport) may not be able to re-use a persistent TCP
+	// connection to the server for a subsequent "keep-alive" request.
+	if respBody != nil {
+		// Drain any remaining Body and then close the connection.
+		// Without this closing connection would disallow re-using
+		// the same connection for future uses.
+		//  - http://stackoverflow.com/a/17961593/4465767
+		defer respBody.Close()
+		io.Copy(io.Discard, respBody)
+	}
```

Comment:
We should limit the size of io.Copy() otherwise it will never end. So we can drain up-to 1MiB